config/daedalus.yml

```diff
@@ -42,4 +42,6 @@ acme_buddy:
     repo: ghcr.io/reside-ic
     name: acme-buddy
     tag: main
+  hdb_username: VAULT:secret/certbot-hdb/credentials:username
+  hdb_password: VAULT:secret/certbot-hdb/credentials:password
```

Comment:
Not sure it's ok for this to be the same for every config, rather than defining this in production.yml and staging.yml. There is a `fakeproxy` config which one ought to be able to run locally without accessing the vault. So I think maybe the values here should be blank, and set to the vault secrets in the two configs. For fakeproxy will correctly set `ssl` to False currently, because proxy host is localhost but it will still attempt to fetch the credentials from the vault. 

In fact, maybe we shouldn't be running acme buddy container at all on localhost..

Reply:
@weshinsley I believe you can have the `acme_buddy` block defined in both files and they will get merged together. That would let you move these to the `production.yml` and `staging.yml` files.